plutus-pab/README.md

```diff
@@ -101,7 +101,7 @@ Generates the purescript bridge code.
 ### migrate
 
 ```
-$ cabal run plutus-pab -- migrate
+$ cabal exec plutus-pab-examples -- migrate --config plutus-pab/plutus-pab.yaml.sample
```

Comment:
Seeing as the readme is in `plutus-pab` directory then config files should be relative to here.
Change this and other occurrences below to `--config plutus-pab.yaml.sample`